src/sotoki/scraper.py

```diff
@@ -77,9 +77,15 @@ def sanitize_inputs(self):
             self.conf.title = Global.site["LongName"]
         self.conf.title = self.conf.title.strip()
 
-        if not self.conf.description:
-            self.conf.description = Global.site["Tagline"]
-        self.conf.description = self.conf.description.strip()
+        default_description = Global.site["Tagline"].strip()
+        if self.conf.description is not None:
```

Comment:
Should be `if self.conf.description` so that we also consider empty string as a description which is not set